CONTRIBUTING.md

```diff
@@ -13,21 +13,27 @@ We're really happy you want to contribute to the project! The following steps wi
 1. Fork and clone the repo;
 2. Install the required dependencies
 
-```sh
-  $ npm install
-```
+   ```sh
+   $ yarn
+   ```
 
-3. Start up the dev server:
+3. Work from within the `packages/core` directory.
 
-```sh
-  $ npm run storybook
-```
+   ```sh
+   $ cd packages/core
+   ```
+
+4. Start up the dev server:
+
+   ```sh
+   $ yarn run storybook
+   ```
```

Comment:
Actually, we have [`yarn dev`](https://github.com/React95/React95/blob/master/package.json#L41) at the root to work with.